site/ohs_configuration.md

```diff
@@ -0,0 +1,49 @@
+## Configuring Oracle HTTP Server (OHS)
+
+Oracle HTTP Server (OHS) can be configured and updated with Create Domain, Update Domain, and Deploy Applications Tools, starting with WDT release 1.8.0. OHS configuration is only supported for offline mode in WDT. The OHS configuration can be discovered using the Discover Domain Tool, in offline mode only.
+ 
+### Prerequisites
+
+In order to configure and use OHS, the product must be installed in the Oracle Home used to create the domain. You can download OHS [here](https://www.oracle.com/middleware/technologies/webtier-downloads.html).
```

Comment:
to configure and use OHS, the product must be installed in the Oracle Home used to create the domain. -> to configure and use OHS, it must be installed in the Oracle Home directory used to create the domain.